### PR TITLE
Array and string offset access syntax with curly braces is deprecated

### PR DESCRIPTION
**Pull request by DeyV**
I know that version 1.0 is not designated for php 7.4 but this deprecation is older .

### CODE REVIEW

**Comment by ste93cry** [comment]
Build is broken due to the fact that PHP < 5.6 is only available in Trusty dist. Can you please add `dist: trusty` to `.travis.yml` and see if it works?

File: lib/Raven/Client.php [hunk 1]
```diff
@@ -328,7 +328,7 @@ private static function _convertPath($value)
         // we need app_path to have a trailing slash otherwise
         // base path detection becomes complex if the same
         // prefix is matched
-        if ($path{0} === DIRECTORY_SEPARATOR && substr($path, -1) !== DIRECTORY_SEPARATOR) {
+        if ($path[0] === DIRECTORY_SEPARATOR && substr($path, -1) !== DIRECTORY_SEPARATOR) {
```
Comment (Jean85) [inline lib/Raven/Client.php]: This may generate a notice with an empty string: https://3v4l.org/GVBHT

Maybe we should use `substr` here too?

Comment (DeyV) [reply lib/Raven/Client.php]: https://3v4l.org/u8HWDh - the same problem with braces

**Comment by stayallive** [comment]
@DeyV sorry about having to push a commit to your branch but to me this was the easiest way to fix a bug that occurs when running with xdebug >=2.8 which your PR uncovered.

Thanks for fixing the deprecations!

**Comment by stayallive** [comment]
`1.11.0` was just release which includes this PR. Thanks again!

**Pull request merged**